tests/hazmat/primitives/utils.py

```diff
@@ -76,6 +76,8 @@ def generate_aead_test(
     param_loader, path, file_names, cipher_factory, mode_factory
 ):
     all_params = _load_all_params(path, file_names, param_loader)
+    # We don't support IVs < 64-bit so just strip them out
+    all_params = [i for i in all_params if len(i["iv"]) >= 16]
```

Comment:
Is this true for all AEADs?

Reply:
No, should be constrained to GCM. Pushing fix